simulation-gazebo.md

```diff
@@ -88,7 +88,7 @@ This will bring up the PX4 shell:
 
 ```sh
 [init] shell id: 140735313310464
-[init] task name: mainapp
+[init] task name: px4
```

Comment:
I think it is easiest to leave it for now so we can minimize changes for the rename, and then remove extra verbose output later if we want.